### DISCUSSION TITLE
Deploy to windows server

### DISCUSSION THREAD

**Discussion by nunosilvafullscreen** [deployphp/deployer#3688] [General]
Hello

I've been using deployer with success on our linux servers, it's been working great!

Unfortunatly we also have some windows servers (hosting), is there any way that deployer can deploy to a windows server?
There are a few potencial problems I'm foreseeing:
1 - creating symlinks in windows
2 - setting file and folder permissions in windows

Is there a way to make deployer deploy to a windows host?
If it's impossible, do you know any good alternatives? 

Thanks

**Comment by antonmedv** [comment]
Deployer it self may work with windows, but not the common recipe. It was designed for linux. Maybe something like ansible for windows will work.

**Comment by nunosilvafullscreen** [comment]
teste
